src/main/java/mate/academy/springboot/datajpa/dto/CategoryRequestDto.java

```diff
@@ -0,0 +1,8 @@
+package mate.academy.springboot.datajpa.dto;
+
+import lombok.Data;
+
+@Data
```

Comment:
Let's not use `@Data` annotation. It's better to use Getter Setter etc. Please fix the same issue in all places